src/functions/postTypes/getFrontendPage.js

```diff
@@ -34,13 +35,18 @@ export default async function getFrontendPage(route) {
       response.defaultSeo = formatDefaultSeoData({ homepageSettings, siteSeo });
 
       // Set route SEO.
+      const canonical = new URL(
+        `${response.defaultSeo?.openGraph?.url ?? ""}/${route}`
+      );
```

Comment:
If I get the code,

```suggestion
      const canonical = new URL(
        `${response.defaultSeo?.openGraph?.url ?? ""}/${route}`
      );
      canonical.pathname = replaceMultisitePrefix(canonical?.pathname);
```

And then

```js
          canonical: `${canonical?.origin}${replaceMultisitePrefix(
            canonical?.pathname
          )}`,
```
would become just
```js
          canonical: canonical.toString(),
```
... unless you're using canonical somewhere else and you don't want it modified.